src/main/java/org/commcare/formplayer/application/FormplayerAuthFilter.java

```diff
@@ -75,10 +72,10 @@ protected void doFilterInternal(HttpServletRequest req, HttpServletResponse resp
                         ace.getResponseCode(),
                         ace.getLog()), ace);
 
-                sentryReporter.sendEvent(
-                    sentryReporter.getEventForException(ace)
-                            .withMessage(ace.getLog())
-                            .withTag("uri",req.getRequestURI()));
+                Sentry.withScope(scope -> {
+                    scope.setTag("uri",req.getRequestURI());
+                    Sentry.captureException(ace);
+                });
```

Comment:
sent Sentry is configured by Spring during startup it is available regardless of whether the application is in a request or not.